From 3542d620a3ea53c93b0c3953df7658874c1e7ee7 Mon Sep 17 00:00:00 2001
From: Sang Ngo <sang.ngo.yf@renesas.com>
Date: Mon, 25 Aug 2025 20:32:10 +0700
Subject: [PATCH] GPU driver: remove power-domains of GPU note for RZV2H

Due to avoid duplicated clock control between the CPG genpd and
the mali-kbase driver, let mali-kbase handle the GPU clocks itself

Signed-off-by: Sang Ngo <sang.ngo.yf@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a09g057.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r9a09g057.dtsi b/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
index e945127e5eb9..bcc8728fc8f6 100644
--- a/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
@@ -2339,7 +2339,6 @@ gpu: gpu@14850000 {
 				 <&cpg CPG_MOD 0xf1>,
 				 <&cpg CPG_MOD 0xf2>;
 			clock-names = "gpu", "bus", "bus_ace";
-			power-domains = <&cpg>;
 			resets = <&cpg 0xdd>,
 				 <&cpg 0xde>,
 				 <&cpg 0xdf>;
-- 
2.25.1

